### PR TITLE
Atiburce/manifest menus and header

### PR DESCRIPTION
**Pull request by adrientiburce**
- I fixed the menus on the left 
- Links to paragraph always go to the top

### CODE REVIEW

File: programmes/renderers/programmes/manifesto_detail.hy [hunk 1]
```diff
@@ -1,28 +1,40 @@
...
 (defn paragraphs [manifesto]
   ['section {'class "container--manifesto-paragraphs"}
     ['div {'class "manifesto-paragraphs"}
       ['h1 "Propositions par thématiques"]
       (gfor paragraph (manifesto.paragraphs.all)
             ['section {'class "manifesto-paragraph"}
-              ['h1 paragraph.topic]
-              (markdown paragraph.text)])]])
+              ['div {'id (slugify paragraph.topic.name)}] ; empty div to have fixed anchor links
+                ['h1 {'id (slugify paragraph.topic.name)} paragraph.topic]
```
Comment (Arkelis) [inline programmes/renderers/programmes/manifesto_detail.hy]: The id on the `h1` is no longer necessary, is it?

File: programmes/static/css/manifesto.scss [hunk 2]
```diff
@@ -6,6 +6,28 @@
   padding: 3rem 0;
 }
 
+.manifesto-menu {
```
Comment (Arkelis) [inline programmes/static/css/manifesto.scss:9]: Font size could be smaller

Comment (adrientiburce) [reply programmes/static/css/manifesto.scss:9]: what size would you like, I neet to import a new google font for that ?

Comment (Arkelis) [reply programmes/static/css/manifesto.scss:9]: No new font needed, just `font-size: .75rem` or something like that

**Review by Arkelis — APPROVED** [review]

**Pull request merged**